redis/NOTES.md

```diff
@@ -0,0 +1,15 @@
+**2024-09-20**
```

Comment:
This is no longer the case since [Unikraft release v0.18.0 (Helene)](https://unikraft.org/releases/latest).